try {
    nonExistentFunction();
} catch (e) {
    console.log('Функция не существует');
}